testah-junit/build.gradle

```diff
@@ -11,7 +11,7 @@ apply plugin: 'jacoco'
 
 sourceCompatibility = 1.8
 group = 'org.testah'
-version = '0.9.9'
+version = '0.9.10'
 

```

Comment:
dont forget the version in the cli class

Reply:
updated pull request